### ISSUE TITLE
Loading a saved file after making chances to a class implementing Serializable fails

### ISSUE THREAD

**Issue by bjornpijnacker** [andrewlalis/EntityRelationMappingEditor#11]
The `serialVersionUID` is not set in the Relation and Attribute classes. Setting the UID in Relation equal to what the error message showed fixed it, so these UIDs should probably be set in any future version to ensure save file compatibility between (minor) versions changes.

**Comment by andrewlalis** [comment]
Yes, that is something I've noticed as well. I will wait until after next week. I plan on serializing the model to a more human-readable format besides the `Serializable` though; I used that because it was quick, but I'm thinking of using JSON or XML as the end game.
